packages/react/src/components/Alert/index.tsx

```diff
@@ -18,13 +18,14 @@ const Alert = ({
   <Dialog
     className={classnames(
       { Alert__warning: variant === 'warning' },
+      { Alert__default: variant === 'default' },
```

Comment:
We can treat the default variant as the variant without a class, so we can remove this line.